protocol/persistence_profile_showcase.go

```diff
@@ -49,7 +93,340 @@ func (db sqlitePersistence) InsertOrUpdateProfileShowcasePreference(entry *Profi
+		asset := &ProfileShowcaseAssetPreference{}
+
+		err := rows.Scan(
+			&asset.Symbol,
+			&asset.ShowcaseVisibility,
+			&asset.Order,
+		)
+
+		if err != nil {
+			return nil, err
+		}
+
+		assets = append(assets, asset)
+	}
+	return assets, nil
+}
+
+// Queries for contacts showcase
+func (db sqlitePersistence) saveProfileShowcaseCommunityContact(tx *sql.Tx, contactID string, community *identity.ProfileShowcaseCommunity) error {
+	_, err := tx.Exec(upsertContactProfileShowcaseCommunityQuery,
+		contactID,
+		community.CommunityID,
+		community.Order,
+	)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db sqlitePersistence) getProfileShowcaseCommunitiesContact(tx *sql.Tx, contactID string) ([]*identity.ProfileShowcaseCommunity, error) {
+	rows, err := tx.Query(selectContactProfileShowcaseCommunityQuery, contactID)
+	if err != nil {
+		return nil, err
+	}
+
+	communities := []*identity.ProfileShowcaseCommunity{}
+
+	for rows.Next() {
+		community := &identity.ProfileShowcaseCommunity{}
+
+		err := rows.Scan(&community.CommunityID, &community.Order)
+		if err != nil {
+			return nil, err
+		}
+
+		communities = append(communities, community)
+	}
+	return communities, nil
+}
+
+func (db sqlitePersistence) clearProfileShowcaseCommunityContact(tx *sql.Tx, contactID string) error {
+	_, err := tx.Exec(removeContactProfileShowcaseCommunityQuery, contactID)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db sqlitePersistence) saveProfileShowcaseAccountContact(tx *sql.Tx, contactID string, account *identity.ProfileShowcaseAccount) error {
+	_, err := tx.Exec(upsertContactProfileShowcaseAccountQuery,
+		contactID,
+		account.Address,
+		account.Name,
+		account.ColorID,
+		account.Emoji,
+		account.Order,
+	)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db sqlitePersistence) getProfileShowcaseAccountsContact(tx *sql.Tx, contactID string) ([]*identity.ProfileShowcaseAccount, error) {
+	rows, err := tx.Query(selectContactProfileShowcaseAccountQuery, contactID)
+	if err != nil {
+		return nil, err
+	}
+
+	accounts := []*identity.ProfileShowcaseAccount{}
+
+	for rows.Next() {
+		account := &identity.ProfileShowcaseAccount{}
+
+		err := rows.Scan(&account.Address, &account.Name, &account.ColorID, &account.Emoji, &account.Order)
+		if err != nil {
+			return nil, err
+		}
+
+		accounts = append(accounts, account)
+	}
+	return accounts, nil
+}
+
+func (db sqlitePersistence) clearProfileShowcaseAccountsContact(tx *sql.Tx, contactID string) error {
+	_, err := tx.Exec(removeContactProfileShowcaseAccountQuery, contactID)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (db sqlitePersistence) saveProfileShowcaseCollectibleContact(tx *sql.Tx, contactID string, community *identity.ProfileShowcaseCollectible) error {
+	_, err := tx.Exec(upsertContactProfileShowcaseCollectibleQuery,
+		contactID,
+		community.UID,
+		community.Order,
+	)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Looks like it can be simplified 😄 
(there're a few such places in this file)
```suggestion
	return err
```